metadata-db-app/src/cmr/metadata_db/services/generic_documents.clj

```diff
@@ -21,53 +21,51 @@
 ;; to database column names which are to be shared with the world.
 (def info-fields [:concept-id :native-id :provider-id :document-name :schema
                   :format :mime-type :revision-id :revision-date :created-at
-                  :user-id])
+                  :user-id :concept-type :concept-sub-type])
 
-;; These are fields that should not be inclucded in the :Metadata field because
-;; they are generated by software. Add items to this list corrisponding to
-;; database column names which should NOT be shared with the world and are not
-;; in the info-fields list above.
-(def fields-to-drop (vec (concat info-fields [:id :deleted :transaction-id])))
+;; Insert-generic-document and other CRUD actions, except Read, return nil on
+;; success, To make the code more understandable, give that success a name.
+;; Fails will be handled with exceptions.
+(def success-response nil)
 
 (defn- raw-generic->response
   "Take the raw database object returned from either SQL or In-Memory and shape
```

Comment:
You either need to delete this function because it isn't used anymore, or (and more preferred) to remove the success-response of nil and return a response from the DB.

Reply:
I deleted this def and return data allowing for a standard response to be built.